.travis.yml

```diff
@@ -20,9 +26,12 @@ jobs:
       env: SCRIPT=test
       # Node 10 instead of 'stable' since Node 11 segfaults
```

Comment:
I would like to erase it if it is stable.

Reply:
@hiroppy let's remove comment and use `node_js: 8`, stable can be changed in future

Reply:
Also let's add node_js: 11

Reply:
Is it like this?(or we should use `'lts/*'`?)

```
    - &osx
      stage: Test (MacOS)
      os: 'osx'
      env: SCRIPT=test
    - <<: *osx
      node_js: 11
    - <<: *osx
      node_js: 10
    - <<: *osx
      node_js: 8
    - <<: *osx
      node_js: 6
```

Reply:
:+1: Let's avoid `lts/*`, i think travis interpreted first line as test

Reply:
Updated